src/sdk/comment.ts

```diff
@@ -0,0 +1,48 @@
+import { Context } from "./context";
+import { LogReturn } from "@ubiquity-os/ubiquity-os-logger";
+import { sanitizeMetadata } from "./util";
+
+const HEADER_NAME = "Ubiquity";
+
+/**
+ * Posts a comment on a GitHub issue if the issue exists in the context payload, embedding structured metadata to it.
+ */
+export async function postComment(context: Context, message: LogReturn) {
+  if ("issue" in context.payload && context.payload.repository?.owner?.login) {
+    const metadata = createStructuredMetadata(message.metadata?.name, message);
```

Comment:
`message.metadata?.name` will require we update the logger. I implemented `pluginName` in this [logger PR](https://github.com/ubiquity-os/ubiquity-os-logger/pull/35/files#diff-9de455cd7a4a742596bdc25ae7ab345f6b402e36d69cdb28d2867193c7a1e5b3R18), is that what `name` represents here?